Uncareted `sanitize.css` dependency causing failure while requiring `postcss-normalize`

[`sanitize.css@13.0.0` removes file `page.css`](https://github.com/csstools/sanitize.css/compare/12.0.0..13.0.0#diff-cf0e02530054350aa3396a6be36d0936e2d8feb37a2f6f6a904616b7afa501d2), which is directly referenced in `cssMap.js`:

https://github.com/csstools/postcss-normalize/blob/5b80f53e46d43d297d7deaec4704c044fa38fb69/src/lib/cssMap.js#L16

As `sanitize.css` is installed at version `*` rather than something like `12.0.0` or `^12.0.0`, `npm` users are auto-upgraded into this breaking `sanitize.css` version. This includes users of [`create-react-app`'s `react-scripts`](https://github.com/facebook/create-react-app/blob/main/packages/react-scripts/package.json#L61).

Can confirm this issue causes `create-react-app` to fail, giving the following error on `start` or `build`:

```
ERROR in ./src/index.css (./node_modules/css-loader/dist/cjs.js??ruleSet[1].rules[1].oneOf[5].use[1]!./node_modules/react-scripts/node_modules/postcss-loader/dist/cjs.js??ruleSet[1].rules[1].oneOf[5].use[2]!./node_module

Loading PostCSS "postcss-normalize" plugin failed: Cannot find module 'sanitize.css/page.css'
Require stack:
- ... \node_modules\react-scripts\node_modules\postcss-normalize\index.cjs
```

--- a/src/lib/cssMap.js
+++ b/src/lib/cssMap.js
@@ -13,7 +13,6 @@ const normalizeCSS = resolve('@csstools/normalize.css')
 const normalizeOpinionatedCSS = resolve('@csstools/normalize.css/opinionated.css')
 const sanitizeCSS = resolve('sanitize.css')
 const sanitizeFormsCSS = resolve('sanitize.css/forms.css')
-const sanitizePageCSS = resolve('sanitize.css/page.css')
 const sanitizeTypographyCSS = resolve('sanitize.css/typography.css')
 
 // export a hashmap of css library filenames
@@ -22,7 +21,6 @@ export const parsableFilenames = create({
 	[normalizeOpinionatedCSS]: true,
 	[sanitizeCSS]: true,
 	[sanitizeFormsCSS]: true,
-	[sanitizePageCSS]: true,
 	[sanitizeTypographyCSS]: true
 })
 
@@ -33,9 +31,9 @@ export const resolvedFilenamesById = create({
 	'normalize/*': [normalizeOpinionatedCSS],
 	'sanitize': [sanitizeCSS],
 	'sanitize/forms': [sanitizeCSS, sanitizeFormsCSS],
-	'sanitize/page': [sanitizeCSS, sanitizePageCSS],
+	'sanitize/page': [sanitizeCSS],
 	'sanitize/typography': [sanitizeCSS, sanitizeTypographyCSS],
-	'sanitize/*': [sanitizeCSS, sanitizeFormsCSS, sanitizePageCSS, sanitizeTypographyCSS]
+	'sanitize/*': [sanitizeCSS, sanitizeFormsCSS, sanitizeTypographyCSS]
 })
 
 // get the resolved filename of a package/module

